torchao/_models/llama/eval.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Thanks! since this is not tested in CI mind just printing the expected output and putting it in the github issue description

Reply:
i added the values to the PR description